drivers/hmis/spec/models/hmis/hud/organization_access_spec.rb

```diff
@@ -61,5 +72,10 @@
       viewable_orgs = Hmis::Hud::Organization.viewable_by(user_with_p2_access)
       expect(viewable_orgs).to contain_exactly(o2)
     end
+
+    it 'includes only organizations associated with the data source ID of the current user' do
```

Comment:
This isn't reproducing the bug, I just added it while I was playing around trying to understand all the pieces, and I figure we can commit it if it's helpful. Happy to revert if not.

Reply:
I think that's a good test case to have. Doesn't look like it's covered by other tests. Thanks!